Src/Microsoft.Dynamic/Generation/FieldBuilderExpression.cs

```diff
@@ -31,9 +31,19 @@ namespace Microsoft.Scripting.Generation {
     public class FieldBuilderExpression : Expression {
         private readonly FieldBuilder _builder;
 
+#if NETCOREAPP2_0
+        private readonly StrongBox<Type> _finishedType;
```

Comment:
The code wouldn't compile since there's no FieldBuilder.GetToken in .NET Core. I just readded the code that was there before it was removed in the Silverlight cleanup.